README.md

```diff
@@ -115,14 +118,12 @@ polaris --audit --audit-path ./deploy/
 
 ##### Running with CI/CD
 You can integrate Polaris into CI/CD for repositories containing infrastructure-as-code.
-For example, to fail whenever the Polaris score drops below 90%:
+For example, to fail whenever the Polaris score drops below 90%, or Polaris detects error-level
```

Comment:
I think this maybe should call out errors first - since it's prioritized in the if logic. I, personally, would also call out:
>"..to fail if Polaris detects _any_ error-level issues, or if there are no error-level issues, fail if the score drops below 90%"

Reply:
👍